### PR TITLE
Create ESEA.net.xml

### PR DESCRIPTION
**Pull request by christianheinrichs**

### CODE REVIEW

File: src/chrome/content/rules/ESEA.net.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+		- esea.net → play.esea.net
+		- news.esea.net → play.esea.net
+-->
+<ruleset name="ESEA.net">
+	<target host="esea.net" />
+	<target host="www.esea.net" />
+	<target host="fragshack.esea.net" />
+	<target host="mdl.esea.net" />
+	<target host="news.esea.net" />
+	<target host="play.esea.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/ESEA.net.xml]: + `static.esea.net` (please also add a valid test url for it)

Comment (christianheinrichs) [reply src/chrome/content/rules/ESEA.net.xml]: @J0WI Done.

File: src/chrome/content/rules/ESEA.net.xml [hunk 2]
```diff
@@ -10,6 +10,8 @@
 	<target host="mdl.esea.net" />
 	<target host="news.esea.net" />
 	<target host="play.esea.net" />
+	<target host="static.esea.net" />
+		<test url="http://static.esea.net/" />
```
Comment (J0WI) [inline src/chrome/content/rules/ESEA.net.xml]: This test url returns a 403 status code. Please change to something that response with 200.

Comment (christianheinrichs) [reply src/chrome/content/rules/ESEA.net.xml]: @J0WI The newest commit on this PR now contains a test URL which should return a 200 status code. From what I understand, the `static.esea.net` is probably exclusively used for the hosting of .cfg files.

**Pull request merged**